views/index.html

```diff
@@ -28,20 +31,13 @@
 
   <body>
     <header>
```

Comment:
Can we just remove lines 33 through 45 for now and replace with:
```
    <h1>MongoDB & Mongoose</h1>
```
Later in a separate PR, we can always add a `header`, `main`, and `footer`, but I can not think of anything to put here and we definitely do not need that footer.